Description: Fix primus $LIB expanding
 In Ubuntu Raring, $LIB expand to lib/arch/, not just arch/.
Origin: upstream, https://github.com/Bumblebee-Project/Bumblebee/commit/36683895d9fce9fa8acad5b83b2a6f020b3f1360
Author: Bruno Pagani <bruno.n.pagani@gmail.com>
Last-Update: 2013-02-25

--- a/src/optirun.c
+++ b/src/optirun.c
@@ -186,7 +186,7 @@ static int run_primus(int argc, char **argv) {
     }
   }
 
-  char *libgl_mesa = "/usr/$LIB/libGL.so.1:/usr/lib/$LIB/libGL.so.1:/usr/lib/$LIB/mesa/libGL.so.1";
+  char *libgl_mesa = "/usr/$LIB/libGL.so.1:/usr/lib/$LIB/libGL.so.1:/usr/$LIB/mesa/libGL.so.1:/usr/lib/$LIB/mesa/libGL.so.1";
   if (bb_config.ld_path[0]) { /* build new library path for PRIMUS_libGLa */
     int libgl_size = strlen(bb_config.ld_path) + 1;
     { /* calculate additional memories for adding "/libGL.so.1" */
